js/sound-player.js

```diff
@@ -0,0 +1,13 @@
+const playerButton = document.querySelector('.player__button');
+const soundItems = document.querySelectorAll('.sounds__item');
+
+// Function that enables/disables a specific sound when it has been clicked
+function handleSoundItemsClick(event) {
+  const soundItem = event.currentTarget;
+  const slider = soundItem.querySelector('.slidecontainer');
```

Comment:
nit-picky comment; we might consider changing that class name to something like:
```suggestion
  const slider = soundItem.querySelector('.slide-container');
```

or

```suggestion
  const slider = soundItem.querySelector('.slide__container');
```